docs/tests/conditionals.rst

```diff
@@ -0,0 +1,209 @@
+Conditional  Statements
+=======================
+
+The conditional section of the pavilion config consists of two main
+directives, ``only_if`` and ``not_if``. This documentation will cover
+how to use them as well as details associated with their use in test
+configs.
+
+Why conditional statements? As Pavilion becomes more popularized as an
+HPC test harness, the test suites it runs will theoretically grow. It is
+unrealistic to manage unique test suites for specific machines. By using
+conditional statements users can have one large test suite that can run
+across all systems. If a user has a test that is specific to a unique
+computer architecture, they can simply add 'only_if arch is unique-arch'.
+Conditional Statements can greatly simplify your testing procedure.
+
+
+only_if
+~~~~~~~
+
+The ``only_if`` section of the pavilion test config allow users to
+specify a series of conditions. As the name would suggest, the test
+will run if and only if ALL conditions are met.
+
+The syntax for writing an ``only_if`` condition is fairly
+straightforward. The ``only_if`` directive is followed by a
+dictionary that should contain at least one key. The key(s)
+should consist of a variable(s) that pavilion can resolve. Below is
+a basic example.
+
+.. code:: yaml
+
+    basic_test:
+        only_if:
+            user: ['calvin']
+        run:
+            cmds:
+                ...
+
+In this example the dictionary following ``only_if`` has a key
+``user`` and the value of 'calvin'. Pavilion will resolve the key and
+check who the actual user is. Pavilion will then compare the resolved
+key with the value and if ``user == calvin`` then the test will run as
+``calvin == calvin``.
+
+Conditional statements can also parse dictionaries with multiple keys
+and values, let's look at a more complicated test.
+
+.. code:: yaml
+
+    advanced_test:
+        only_if:
+            user: ['calvin', 'paul']
+            sys_os: ['linux']
+            weekday: ['Monday', 'Wednesday', 'Friday']
+        run:
+            cmds:
+                ...
+
```

Comment:
Just give the example above two 'only_if' conditions, and it will be clear that this is the case.